### PR TITLE
Update release notes draft to version v1.22.0-beta.0

### PR DESCRIPTION
**Pull request by cici37**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
This PR updates the Release Notes Draft to k/k v1.22.0-beta.0

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:
This is an automated PR generated from `krel The Kubernetes Release Toolbox`

### CODE REVIEW

**Comment by cici37** [comment]
/assign @puerco @pmmalinov01 

cc @simcard0000 @Damans227 @sladyn98

File: releases/release-1.22/release-notes/maps/pr-101030-map.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+pr: 101030
+releasenote:
+  text: The pod resource API will provide memory manager metrics in the case when
+    the memory manager feature gate enabled and the memory manager policy is static.
```
Comment (pmmalinov01) [inline releases/release-1.22/release-notes/maps/pr-101030-map.yaml:4]:
```suggestion
    the memory manager feature gate is enabled, and the memory manager policy is static.
```

File: releases/release-1.22/release-notes/maps/pr-101294-map.yaml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+pr: 101294
+releasenote:
+  text: The `SetHostnameAsFQDN` has been graduated to GA, which means feature cannot
```
Comment (pmmalinov01) [inline releases/release-1.22/release-notes/maps/pr-101294-map.yaml:3]:
```suggestion
  text: The `SetHostnameAsFQDN` graduates to GA and thus will be unconditionally enabled.
```

File: releases/release-1.22/release-notes/maps/pr-102731-map.yaml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+pr: 102731
+releasenote:
+  text: The `endpointslicemirroring` controller no longer mirrors the `last-applied-configuration`
```
Comment (pmmalinov01) [inline releases/release-1.22/release-notes/maps/pr-102731-map.yaml:3]:
```suggestion
  text: The `EndpointSliceMirroring ` controller no longer mirrors the `last-applied-configuration`
```

File: releases/release-1.22/release-notes/maps/pr-102731-map.yaml [hunk 4]
```diff
@@ -0,0 +1,4 @@
...
+    annotation created by `kubectl` to update endpoint slices.
```
Comment (pmmalinov01) [inline releases/release-1.22/release-notes/maps/pr-102731-map.yaml:4]:
```suggestion
    annotation created by `kubectl` to update `EndpointSlices`.
```

**Comment by pmmalinov01** [comment]
@cici37 Thank you and great job left a few small remarks. 
I will place it on hold just in case.
/hold

File: releases/release-1.22/release-notes/maps/pr-102873-map.yaml [hunk 5]
```diff
@@ -0,0 +1,5 @@
+pr: 102873
+releasenote:
+  text: Deprecate `--record` flag in `kubectl`. The `--record` flag is being replaced
+    with the [mechanism](https://github.com/kubernetes/enhancements/tree/master/keps/sig-cli/859-kubectl-headers)
+    which annotates HTTP requests with kubectl command details.
```
Comment (Damans227) [inline releases/release-1.22/release-notes/maps/pr-102873-map.yaml:5]: Should it be - which annotates HTTP requests with `kubectl` command details ?

**Review by Damans227 — CHANGES_REQUESTED** [review]
Added my review, lemme know if it helps.

File: releases/release-1.22/release-notes/maps/pr-100183-map.yaml [hunk 6]
```diff
@@ -0,0 +1,3 @@
+pr: 100183
+releasenote:
+  text: Fixed starting new pods after previous pod timed out unmounting its volumes.
```
Comment (Damans227) [inline releases/release-1.22/release-notes/maps/pr-100183-map.yaml:3]:
```suggestion
  text: Fixed starting new `pods` after previous `pod` timed out while un-mounting its volumes.
```

File: releases/release-1.22/release-notes/maps/pr-102731-map.yaml (hunk 3 above)
Comment (cici37) [reply releases/release-1.22/release-notes/maps/pr-102731-map.yaml]: I will go with `EndpointSlice Mirroring Controller` as in https://discuss.kubernetes.io/t/last-week-in-kubernetes-development-week-ending-july-12-2020/11845

File: releases/release-1.22/release-notes/maps/pr-102873-map.yaml (hunk 5 above)
Comment (cici37) [reply releases/release-1.22/release-notes/maps/pr-102873-map.yaml:5]: kubectl does not need to be quoted. ref: releases/release-1.21/release-notes/maps/pr-99833-map.yaml

File: releases/release-1.22/release-notes/maps/pr-100183-map.yaml (hunk 6 above)
Comment (cici37) [reply releases/release-1.22/release-notes/maps/pr-100183-map.yaml:3]: same as kubectl

**Pull request merged**